Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
+      event_sourcery (>= 0.14.0)
       pg
-      sequel (~> 4.38)
-    highline (1.7.8)
-    method_source (0.8.2)
-    mustermann (1.0.0)
+      sequel (>= 4.38)
+    highline (2.0.0)
+    method_source (0.9.2)
+    mustermann (1.0.3)
     pg (0.20.0)
-    pry (0.10.4)
+    pry (0.12.2)
       coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
-    pry-byebug (3.4.2)
-      byebug (~> 9.0)
+      method_source (~> 0.9.0)
+    pry-byebug (3.6.0)
```

Comment:
### [pry-byebug](https://github.com/deivid-rodriguez/pry-byebug)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.4.2 → 3.6.0

[_~~change-log~~, [source-code](https://github.com/deivid-rodriguez/pry-byebug)_]





<details>
<summary>Commits</summary>

A change of **72** commits. See the full changes on [the compare page](https://github.com/deivid-rodriguez/pry-byebug/compare/v3.4.2...v3.6.0).

These are the first 10 commits:
- (25b0c1b) [Docs: Clarify installation](https://github.com/deivid-rodriguez/pry-byebug/commit/25b0c1b9aede05277d12628986a5474023ede050)
- (562cc08) [Merge pull request #111 from jaredbeck/patch-1](https://github.com/deivid-rodriguez/pry-byebug/commit/562cc0888e42c55a349971f46688eb1717d91c2f)
- (7a46c1e) [Use the latest Rubies on Travis CI](https://github.com/deivid-rodriguez/pry-byebug/commit/7a46c1e22fba355bb9734d1f6ac7a7f1de40b6af)
- (a1264e2) [Merge pull request #114 from hisas/ci_against_latest_rubies](https://github.com/deivid-rodriguez/pry-byebug/commit/a1264e21d358a09a9c02f910138afe19be64f2cb)
- (30ee2a8) [Bump chandler from 0.6.0 to 0.7.0](https://github.com/deivid-rodriguez/pry-byebug/commit/30ee2a8ecf14e07b9965174847a944390978cecc)
- (689fa38) [Merge pull request #115 from deivid-rodriguez/dependabot/bun…](https://github.com/deivid-rodriguez/pry-byebug/commit/689fa38bbeb34d0f7e0191aa9250133241342113)
- (18edcd6) [Bump overcommit from 0.37.0 to 0.40.0](https://github.com/deivid-rodriguez/pry-byebug/commit/18edcd6871f05cba774e73d54ac095a3e5c1686f)
- (61e223c) [Merge pull request #118 from deivid-rodriguez/dependabot/bun…](https://github.com/deivid-rodriguez/pry-byebug/commit/61e223c9401ef83f56a8469578e94c97fe24cfe6)
- (9717ae3) [Bump rubocop from 0.46.0 to 0.49.1](https://github.com/deivid-rodriguez/pry-byebug/commit/9717ae3ebd0ea4ccbd451ab9052339ea192b7144)
- (e54493a) [Merge pull request #119 from deivid-rodriguez/dependabot/bun…](https://github.com/deivid-rodriguez/pry-byebug/commit/e54493a17955a41146f73df8cce90a52f27fbfbd)


</details>